ntoskrnl/include/internal/ob.h

```diff
@@ -58,6 +58,11 @@
 //
 #define ObpAccessProtectCloseBit 0x02000000L
 
+//
+// System Process Quota Block
+//
+#define OBJ_SYSTEM_PROCESS_QUOTA (PEPROCESS_QUOTA_BLOCK)1
```

Comment:
```suggestion
#define OBJ_SYSTEM_PROCESS_QUOTA    ((PEPROCESS_QUOTA_BLOCK)1)
```
The "1" value might also require an extra `(ULONG_PTR)` cast (before the pointer one)...